tests/brevitas/graph/test_channel_splitting.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (C) 2023, Advanced Micro Devices, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+import torch
+
+from brevitas.fx import symbolic_trace
+from brevitas.graph.channel_splitting import _clean_regions
+from brevitas.graph.channel_splitting import _split
+from brevitas.graph.equalize import _extract_regions
+from brevitas.graph.fixed_point import MergeBatchNorm
+
+from .equalization_fixtures import *
+
+SPLIT_RATIO = 0.1
+
+
+@pytest.mark.parametrize('split_input', [False, True])
+def test_toymodels(toy_model, split_input, request):
+    test_id = request.node.callspec.id
+
+    torch.manual_seed(SEED)
+
+    model_class = toy_model
+    model = model_class()
+    if 'mha' in test_id:
+        inp = torch.randn(IN_SIZE_LINEAR)
+    else:
+        inp = torch.randn(IN_SIZE_CONV)
+
+    model.eval()
+    expected_out = model(inp)
+
+    model = symbolic_trace(model)
+    # merge BN before applying channel splitting
+    model = MergeBatchNorm().apply(model)
+
+    # save model's state dict to check if channel splitting was done or not
+    old_state_dict = model.state_dict()
+
+    regions = _extract_regions(model)
+    regions = _clean_regions(regions)
+    if not len(regions) > 0:
+        pytest.skip(reason='No regions supported.')
```

Comment:
We should check that len(regions) == 0 for only the toymodels we expect it to be zero